features/step_definitions/work_steps.rb

```diff
@@ -373,11 +373,12 @@
 end
 
 # Posts a chapter for the current user
-When /^I post a chapter for the work "([^"]*)"$/ do |work_title|
+When /^I post a chapter for the work "([^"]*)"(?: as "(.*?)")?$/ do |work_title, pseud|
```

Comment:
You can use the `# rubocop:disable Cucumber/RegexStepName` comment to disable rubocop for this step